packages/slice-machine/src/modules/selectedSlice/sagas.ts

```
@@ -56,6 +58,9 @@ export function* generateSliceScreenshotSaga({
         imageLoading: false,
       });
     }
+
+    void Tracker.get().trackScreenshotTaken({ type: "automatic" });
```

should we care if it fails and we stick to a silent error here? and same below

---

There's a warning inside of the tracker :)

---

yeah I saw, I was just wondering if we wanted a feedback on our side and not on the client side but I'm fine with it